postreise/analyze/generation/capacity_value.py

```diff
@@ -57,3 +61,39 @@ def calculate_net_load_peak(scenario, resources, hours=100):
     net_demand = total_demand - resource_generation
     top_hours = net_demand.sort_values(ascending=False).head(hours).index
     return resource_generation[top_hours].mean()
+
+
+def get_capacity_by_resources(scenario, area, resources, area_type=None):
+    """Get total capacity value of certain resources in the specific area of a
```

Comment:
I would rephrase "capacity value" as this might be misinterpreted as something besides the nameplate capacity

Reply:
`get_nameplate_capacity` ?

Reply:
I think the function name is fine, I was talking about the docstring. "capacity" by itself doesn't imply the same thing as "capacity value".